resources/examples/manager/SampleAssetManager/python/SampleAssetManager/SampleAssetManagerInterface.py

```diff
@@ -0,0 +1,40 @@
+#
+#   Copyright 2013-2021 [The Foundry Visionmongers Ltd]
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+
+# pylint: disable=missing-module-docstring
+
+from openassetio.managerAPI import ManagerInterface
+
+__all__ = ['SampleAssetManagerInterface', ]
+
+# As we are building out the implementation vertically, we have known
```

Comment:
[consider] Prefix with a `TODO` so we can find such things easily later.  One trick I've used previously is to have some tag e.g.
```
# TODO(DF): @pylint-disable
```
or something similar, to help grepping in the future.